<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalRegistrationDialog</class>
 <widget class="QDialog" name="GlobalRegistrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Registration Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Cycle List</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="circleListWidget"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="0">
            <widget class="QPushButton" name="consolidatePushButton">
             <property name="text">
              <string>Consolidate</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Sort</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="initializeCyclePushButton">
             <property name="text">
              <string>Initialize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="addPushButton">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="../res/Registar.qrc">
               <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="deletePushButton">
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset resource="../res/Registar.qrc">
               <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="closePushButton">
             <property name="text">
              <string>Close</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="consistentPushButton">
             <property name="text">
              <string>Non-uniform Consistent</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Uniform Consistent</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Pair List</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="pairListWidget">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="initializePairPushButton">
             <property name="text">
              <string>Initialize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="deinitializePairPushButton">
             <property name="text">
              <string>Deinitialize</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="8" column="1">
        <widget class="QLineEdit" name="axisLineEdit"/>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="angleLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QTextEdit" name="transformationTextEdit"/>
       </item>
       <item row="5" column="0" rowspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Transformation: T1 * T2 * ... * Tn</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="cycleComboBox"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="translationLineEdit"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Translation : R1*...*Rn-1*tn+...+R1*t2+t1 </string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="scaleLineEdit"/>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Rotation : R1 * R2 * ... * Rn</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="ovlNumber1LineEdit"/>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="error1LineEdit"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Scale : c1 * c2 *...* cn</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" rowspan="2">
        <widget class="QTextEdit" name="rotationTextEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Axis</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Angle</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Overlapping Number 1</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Accumulated RMS error 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="selectPushButton">
         <property name="text">
          <string>Select Best</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="sendRelationPushButton">
         <property name="text">
          <string>Send To Tranformations Relationship</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="estimatePushButton">
         <property name="text">
          <string>Estimate</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Accumulated RMS error 2</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLineEdit" name="ovlNumber2LineEdit"/>
       </item>
       <item row="15" column="1">
        <widget class="QLineEdit" name="error2LineEdit"/>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Overlapping Number 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Transformations Relationship</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="relationTextEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="exportPushButton">
       <property name="text">
        <string>Export Transformations</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/Registar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>GlobalRegistrationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
